src/ui/components/Sidebar.tsx

```diff
@@ -1,28 +1,54 @@
 import { FC } from "react";
 import { Link } from "react-router-dom";
+import { Nav } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
 
 const Sidebar: FC = () => {
   return (
-    <div className="sidebar">
-      <Link to="/experiment-json" className="sidebar__link">
+    <Nav className="sidebar rounded position-fixed m-2 light-background">
+      <Nav.Link
+        as={Link}
+        to="/experiment-json"
+        className="sidebar__link fw-medium p-4 dark-font"
```

Comment:
Do we still need the `sidebar__link` class?  Could it be written using bootstrap classes?